printer/src/algorithm.rs

```diff
@@ -78,6 +78,8 @@ pub struct Printer {
     indent: usize,
     // Buffered indentation to avoid writing trailing whitespace
     pending_indentation: usize,
+    // To correctly print with dos line endings(CRLF)
+    dos_line_endings: bool,
```

Comment:
How about making `CRLF` more explicit by renaming to:
```suggestion
    crlf_line_endings: bool,
```